packages/apiserver/src/resources/item/item.resolver.ts

```diff
@@ -3,6 +3,10 @@ import { BookInput, ProlongTimeInput } from '@mimir/global-types';
 import { ItemService } from './item.service';
 import { Status } from '../statuses/status.entity';
 
+class ItemError {
+  constructor(public message: string) {}
+}
+
```

Comment:
can you extend from a specific base nest error class?

Reply:
No, it won't work in this case.
It's a datatype defined in graphql schema. If I return throwable error, graphql will send it as an error instead of data. And our frontend expects error as data type